Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -600,8 +600,8 @@ private async Task<bool> CanUpdateAsync(string resource)
                             if (newsItem.Accepted != false)
                                 return false;
 
-                            // can update if user is admin
-                            if (user_groups.Contains(AuthGroup.SiteAdmin))
+                            // can update if user is admin or SNAdmin
+                            if (user_groups.Contains(AuthGroup.SiteAdmin) || user_groups.Contains(AuthGroup.NewsAdmin))
```

Comment:
Is AuthGroups.NewsAdmin already defined? I would have thought you would have to map it to an Active Directory security group.

Reply:
Yes, I have checked the AuthGroup. The NewsAdmin is in there.

Reply:
Interesting, ok!